src/lib.rs

```diff
@@ -496,28 +506,29 @@ pub fn run(args: &Arguments, mut tests: Vec<Trial>) -> Conclusion {
         }
     } else {
         // Run test in thread pool.
-        let pool = match args.test_threads {
-            Some(num_threads) => ThreadPool::new(num_threads),
-            None => ThreadPool::default()
-        };
+        let num_tests = tests.len();
         let (sender, receiver) = mpsc::channel();
 
-        let num_tests = tests.len();
```

Comment:
Can you move the `let num_tests` line to its previous position (right above the loop) to reduce the diff size?